conf/base.config

```diff
@@ -73,10 +75,14 @@ process {
   withName:markDuplicates {
     cpus = { check_max( 2 * task.attempt, 'cpus' ) }
     memory = { check_max( 16.GB * task.attempt, 'memory' ) }
+    errorStrategy = { task.exitStatus in [1,143,137] ? 'retry' : 'terminate' }
```

Comment:
As above, this isn't needed if we update the line that applies to all processes. We should keep the line below though, as 3 is different to the default.